lib/dom-patcher.js

```diff
@@ -33,6 +33,7 @@ export class DOMPatcher {
     this.state = Object.assign({}, initialState);
     this.renderFunc = renderFunc;
     this.vnode = this.renderFunc(this.state);
+    this.component = options.component;
```

Comment:
alternatively, could pass through the threshold and name directly and dispatch the event on the window.